build/helper/documentation_helper.py

```diff
@@ -522,6 +562,7 @@ def assert_rst_strings_are_equal(expected, actual):
         'GetTurtleID': {
             'codegen_method': 'public',
             'returns': 'ViStatus',
+            'method_templates': [{'filename': '/default_method', 'suffix': '', }, ],
```

Comment:
In this context, one would think 'suffix' refers to the method template.
It actually refers to a suffix to be added to to the python_name for the method.

key should be renamed ('method_python_name_suffix'?), and plase add info about this to https://github.com/ni/nimi-python/wiki/API-Metadata

Reply:
Changed code to use 'method_python_name_suffix'

I will add documentation for 'method_templates' to that page.